components/root/root.test.jsx

```diff
@@ -79,6 +79,7 @@ describe('components/Root', () => {
 
         wrapper.instance().onConfigLoaded();
         expect(props.history.push).toHaveBeenCalledWith('/signup_user_complete');
+        wrapper.unmount();
```

Comment:
Why are you unmounting the wrapper at the end of these tests? I didn't think that was necessary to clean up Enzyme.

Reply:
This was added because location reload was called 9 times as there are 9 tests. If I don't call unmount on the wrapper then componentWillUnmount won't exexcuted and so window event listener wasn't removed. So there will be 9 listeners because each test case is adding the listener on storage event

Reply:
This is also only to support requirements to only auto login on focus, if we just do immediate login then these unmount calls are not required.